h/auth/policy/_identity_base.py

```diff
@@ -16,6 +16,16 @@ def identity(cls, request) -> Optional[Identity]:  # pylint: disable=unused-argu
         """
         return None
 
+    def permits(self, request, context, permission) -> bool:
+        """
+        Get whether a given request has the requested permission on the context.
+
+        :param request: Pyramid request to extract identity from
+        :param context: A context object
+        :param permission: The permission requested
+        """
+        return identity_permits(self.identity(request), context, permission)
```

Comment:
We aren't making use of this yet, but this adds the Pyramid 2.0 security policy interface to this object. That means _as far as our policy interface alone_ is concerned we are now Pyramid 2.0 compatible. 

There are plenty of other things which need to be removed and fixed no doubt before we can go, but this is the next step.

This also nicely demonstrates how the whole system will hang together in the future. The difference will be that `identity_permits` won't end up using ACL stuff internally.

Reply:
Yeah I can't remember why this was in here. There probably was a thought process, but I can't remember it. I've split this off into a separate PR: https://github.com/hypothesis/h/pull/7027